graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -631,8 +667,152 @@ def test_process_edge_labels_link_prediction(
         spark, dghl_loader, label_metadata_dicts
     )
 
+    ensure_masks_are_correct(train_mask_df, val_mask_df, test_mask_df, NUM_DATAPOINTS, split_rates)
+
+
+def test_process_edge_labels_multitask(
+    spark: SparkSession, dghl_loader: DistHeterogeneousGraphLoader
+):
+    """Test processing multi-task link prediction and regression edge labels"""
+    edges_df = create_edges_df_num_label(spark, 0)
+    lp_split_rates = {"train": 0.6, "val": 0.2, "test": 0.2}
+    reg_split_rates = {"train": 0.8, "val": 0.1, "test": 0.1}
+
+    lp_mask_names = [
+        "train_mask_lp",
```

Comment:
You also need to add a test like

lp_mask_names = ["lp_train_mask", "lp_val_mask", "lp_test_mask"]